### PR TITLE
fullstatck-shortlink-by-chenxin

### PR DESCRIPTION
**Pull request by imxinz**
Project：Shortlink 
Desc：The shortlink project is developed with fullstack.
Core：midway2&hooks&express4 + mysql + vite + jest
Test Coverage：95.91%
Author：xinz（陈新）

### CODE REVIEW

File: fullstack/chenxin/shortlink/src/apis/config/config.default.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  "Z",
+];
+
+// ORM配置
+export const orm = {
+  type: "mysql",
+  host: "localhost",
+  port: 8889,
+  username: "root",
+  password: "1qaz@WSX",
```
Comment (ctennis) [inline fullstack/chenxin/shortlink/src/apis/config/config.default.ts:81]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

Comment (imxinz) [reply fullstack/chenxin/shortlink/src/apis/config/config.default.ts]: thanks for reminding. i updated the code, pls review it. thanks again.

**Comment by imxinz** [comment]
thanks for reminding. i updated the code, pls review it. thanks again.

File: fullstack/chenxin/shortlink/.env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+MYSQL_USERNAME=root
+MYSQL_PASSWORD=1qaz@WSX
```
Comment (ctennis) [inline fullstack/chenxin/shortlink/.env:2]: :exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

**Comment by imxinz** [comment]
增加前台界面

**Pull request closed without merging**